source/dlangbot/database.d

```diff
@@ -0,0 +1,54 @@
+/**
+Implements a database to keep track of the points scored
+by authors that fix bugs. The table has the following
+columns:
+
+Github account (Primary key)
+Total points
+*/
+module dlangbot.database;
+
+import ae.sys.database : Database;
+
+private Database _db;
+enum table_schema = q"SQL
+CREATE TABLE [LEADERBOARD] (
+[AUTHOR]  TEXT PRIMARY KEY NOT NULL,
+[POINTS]  INTEGER
```

Comment:
SQL is not case sensitive, so I suggest taking advantage of that and using different case for different parts of statements to improve readability. This initial commit will also set the convention for future additions.

In my code I use upper case for the SQL language, and pascal case for identifiers (table and column names).